src/_guides/language/effective-dart/design.md

```diff
@@ -1087,8 +1354,8 @@ void log(Object object) {
   print(object.toString());
```

Comment:
_Maybe_:

```dart
void log(Object o) => print(o.toString());
```

Reply:
Hmm, I know that's a thing that works now, but I don't find it particularly idiomatic outside of setters and don't really want to encourage it.

Reply:
Ok